frontend/Checkout/components/Item/components/Product/components/Layout/components/ProductQuantity/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './style'
+
+const ProductQuantity = ({ quantity }) => (
+  <div className={styles.quantity}>
+    <span>quantity: {quantity}</span>
```

Comment:
It would be better to use a translation string here already, as it is likely to be needed in the final implementation as well.